pyproject.toml

```diff
@@ -0,0 +1,44 @@
+###############################################################################
+# mpi-sppy: MPI-based Stochastic Programming in PYthon
+#
+# Copyright (c) 2024, Lawrence Livermore National Security, LLC, Alliance for
+# Sustainable Energy, LLC, The Regents of the University of California, et al.
+# All rights reserved. Please see the files COPYRIGHT.md and LICENSE.md for
+# full copyright and license information.
+###############################################################################
+
```

Comment:
```suggestion
```
wrong header...

Reply:
Oops! Sorry, this was an embarrassing copy/paste oversight.